### PR TITLE
Adding the Projects, Contact and Footer section.

### PR DESCRIPTION
**Pull request by Timothy-Achonu**

### CODE REVIEW

File: src/Pages/Projects/Projects.js [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+      <p className={styles.projectPara}>
+        Here are a few projects we have worked on both individually and
+        collaboratively.
+      </p>
+     
+         <Pagination data={data}
+         />
+    </div>
+  );
+};
```
Comment (Tijani-zainab) [inline src/Pages/Projects/Projects.js:24]: Hey You @Timothy-Achonu 
don't know if it was intentional, but you're not exporting some of your components like this one.

File: src/Pages/Projects/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
```
Comment (Tijani-zainab) [inline src/Pages/Projects/Pagination.js:1]: also, Nice work with the pagination !
I got lost as to why we needed pagination on that section though, you probably make the good call

Nice work ! @Timothy-Achonu

**Review by Tijani-zainab — APPROVED** [review]

File: src/Pages/Footer/Footer.js [hunk 3]
```diff
@@ -1,14 +1,47 @@
...
+          </div>
+          <div className={styles.socials}>
+            <span>f</span>
+            <span>T</span>
+            <span>I</span>
+            <span>L</span>
+          </div>
+        </div>
+        <div className={styles.footerContactUs}>
+          <div>Contact Us</div>
```
Comment (Tijani-zainab) [inline src/Pages/Footer/Footer.js:24]: I was also lost in the footer section, cos I'm not sure this is traditional functional/arrow function component, but if it works.
Good work !

Probably some new thing I don't know hehe.
Nice work again, can't wait till you review my own work !

**Review by Tijani-zainab — APPROVED** [review]

File: src/Pages/Projects/Pagination.js (hunk 2 above)
Comment (Timothy-Achonu) [reply src/Pages/Projects/Pagination.js:1]: Thank you.

**Pull request merged**